dask/array/tests/test_overlap.py

```diff
@@ -294,6 +294,11 @@ def test_asymmetric_overlap_boundary_exception():
 
 
 def test_map_overlap():
+    x = da.from_array(np.arange(10), chunks=5)
+    y = x.map_overlap(lambda x: x + 1, depth=0)
+    assert len(y.dask) == 2 * x.numblocks[0]  # depth=0 --> map_blocks
```

Comment:
This might be overly precise, but I don't have a firm opinion.

Reply:
I think this is the most stable thing to check, and so I'm in favour of being precise here. 

This way we only make assumptions about the number of tasks in `map_blocks`, rather than the number (or relative numbers) of tasks in `map_blocks` *and* `map_overlap`.